products/sle15/profiles/pci-dss.profile

```diff
@@ -62,8 +66,10 @@ selections:
     - audit_rules_usergroup_modification_passwd
     - audit_rules_usergroup_modification_shadow
     - audit_rules_sysadmin_actions
+    - dconf_gnome_screensaver_idle_activation_enabled
```

Comment:
if your really want this rule selected here, first you have to add `sle15` to the `prodtype` attribute here: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_screensaver_idle_activation_enabled/rule.yml#L3

otherwise the build will always fail

Reply:
or maybe you want to remove it from here since you already have it selected here: https://github.com/ComplianceAsCode/content/pull/7338

Reply:
Yup! Add here accidentally, it is in the other PR you mentioned. Removing it from this one. thanks!